index.js

```diff
@@ -59,9 +59,15 @@ async function getJobStatus(jobName) {
     );
 }
 async function waitJenkinsJob(jobName, timestamp) {
+  let buildNumber = undefined
   core.info(`>>> Waiting for "${jobName}" ...`);
+  await sleep(10) // wait for the new job to show up under lastBuild
```

Comment:
Is this code for testing? or intended? 
I think it doesn't need to wait 10 secs whenever create new jenkins job. The time it takes for a new job to be created will vary depending on the Jenkins machine and environment. (actually it's very fast on my machine.) 
So it is okay to pull jobs right away and check the latest job is created after the time the request sent.